selinux/yggdrasil.te

```diff
@@ -0,0 +1,41 @@
+policy_module(yggdrasil, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type yggd_t;
+type yggd_exec_t;
+init_daemon_domain(yggd_t, yggd_exec_t)
+
+permissive yggd_t;
+
+########################################
+#
+# yggd local policy
+#
+allow yggd_t self:capability { setgid setuid };
+allow yggd_t self:fifo_file rw_fifo_file_perms;
+allow yggd_t self:unix_stream_socket create_stream_socket_perms;
+allow yggd_t self:unix_dgram_socket { connect create getattr };
+
+domain_use_interactive_fds(yggd_t)
+
+files_read_etc_files(yggd_t)
+
+files_watch_etc_files(yggd_t)
```

Comment:
I have just note that is not related to this case. We had to consider use case for rhsm.service, when some customers use NFS for sharing `/etc` directory and it wasn't possible to use i-notify. We had to implement regular polling as fallback solution. I hope such request will not pop up for any client tool anymore.

Reply:
That's alarming. Machine-specific information goes into `/etc`. Sharing it over NFS seems unsafe. I feel like that's a condition that we should not support unless we're explicitly required to. That introduces a lot of complexity unnecessarily.